examples/android/sync/app/src/androidTest/java/com/mongodb/realm/examples/kotlin/ErrorHandlerTest.kt

```diff
@@ -0,0 +1,46 @@
+package com.mongodb.realm.examples.kotlin
+
+import android.util.Log
+import com.mongodb.realm.examples.RealmTest
```

Comment:
Your tests are failing because you never import the expectation class.
```suggestion
import com.mongodb.realm.examples.RealmTest
import com.mongodb.realm.examples.Expectation
```

Reply:
Fixed